Add a Python-specific landing page

ericholscher opened the pull request:
This reuses a lot of content but also tries a few new approaches.

## Preview

![127 0 0 1_8000_publishers_python_](https://user-images.githubusercontent.com/25510/127690316-a35c59d7-5aa1-4cb8-be30-6841b78fc83c.png)

davidfischer commented on content/pages/landing-pages/python.md:
```diff
@@ -0,0 +1,6 @@
+title: Privacy-first ads on Python projects
+slug: python
+url: publishers/python/
```
Just a thought: Should it be `/publishers/topics/python/`?

davidfischer commented on ethicalads-theme/templates/ea/landing-pages/python.html:
```diff
@@ -0,0 +1,270 @@
+
+      <div class="col-md-6 mb-3">
+        <h2 class="font-weight-bold mb-8">Python-focused ads</h2>
+        <p class="font-size-lg text-gray-800">We use the content of your pages to target advertising, so you'll only get ads that are relevent to your content. <strong>No ads that track your users around the internet.</strong> You can see every advertiser that shows an ad on your site.</p>
```
Technically they only see the top 20 but I guess it's a fine claim.

ericholscher replied:
Said `major` 👍

davidfischer reviewed (approved):
This looks like a good start. Where should we link to this from? It would be nice to ensure it's indexed.

ericholscher commented:
@davidfischer addressed feedback. Also rejiggered the footer to make it a bit more balanced:

<img width="984" alt="Screen Shot 2021-08-02 at 4 11 14 PM" src="https://user-images.githubusercontent.com/25510/127934659-4667ddb3-2d9c-42dd-9b51-8f8eaa7a2200.png">

davidfischer commented:
:ship: Ship it!
